huggingface/pytorch/buildspec-trcomp.yml

```diff
@@ -24,7 +24,7 @@ images:
     tag_python_version: &TAG_PYTHON_VERSION py38
     cuda_version: &CUDA_VERSION cu113
```

Comment:
Base PT just launched https://github.com/aws/deep-learning-containers/pull/1888. We need to check what https://github.com/aws/deep-learning-containers/pull/1824 - do we need to support both 113 and 115 ? or are we skipping 113 in favor of 115 ?

Reply:
Hey Loki, how about we support 113 first , since we already have it ready and later after this PR is merged we can support 115 as well.

Reply:
We cannot merge this PR before https://github.com/aws/deep-learning-containers/pull/1824. I would suggest we wait and see what HF does. It's best if our version matches exactly what HF releases - nothing more and nothing less for the sake of fungibility.